config/application.yml.example

```diff
@@ -1,45 +1,117 @@
-test:
-  DASHBOARD_SECRET_KEY_BASE: 'changeme'
-  IDP_SLO_URL: 'http://idp.example.com/saml/logout'
-  IDP_SP_URL: 'http://idp.example.com/api/service_provider'
-  IDP_SSO_URL: 'http://idp.example.com/saml/auth'
-  LOGIN_DASHBOARD_TOKEN: 'sekret'
-  NEW_RELIC_LICENSE_KEY: 'changeme'
-  SMTP_ADDRESS: 'changeme'
-  SMTP_DOMAIN: 'changeme'
-  SMTP_PASSWORD: 'changeme'
-  SMTP_USERNAME: 'changeme'
-  SP_NAME: 'changeme'
-  SP_PASS: 'changeme'
+development: &default
+  admin_email: 'partners@login.gov'
+  basic_auth_username: ''
+  basic_auth_password: ''
+  dashboard_api_token: 'sekret'
+  idp_sp_url: 'http://localhost:3000/api/service_provider'
+  mailer_domain: 'https://dashboard.login.gov'
+  newrelic_license_key: ''
+  saml_idp_fingerprint: '8B:D5:C2:E8:9A:2B:CE:B7:4B:95:50:BA:16:79:05:27:17:D1:D3:67'
+  saml_idp_sso_url: 'http://localhost:3000/api/saml/auth'
+  saml_idp_slo_url: 'http://localhost:3000/api/saml/logout'
+  saml_name_id_format: 'urn:oasis:names:tc:SAML:1.1:nameid-format:persistent'
+  saml_sp_issuer: 'https://dashboard.login.gov'
+  saml_sp_certificate: |
```

Comment:
would it simplify things on the chef side if we made these paths to files on the filesystem instead?